apps/site/src/_pages/docs/2_working-with-types.mdx

```diff
@@ -43,7 +43,7 @@ You can also define the _links_ your entity expects following a similar process:
 1.  Start typing to see if a suitable link entity type already exists (e.g. called 'Owner')
 2.  Either select an existing link entity type, or click 'create new' to create a new one
 3.  When creating a link entity type, you first specify a title and description
-4.  The expected _destination_ entity types for a given link are specified on each entity type it appears on
+4.  Back on your initial entity type, you can use the link entity type, and then constrain the _destination_ entities of those links by defining a selection of entity types
```

Comment:
```suggestion
4.  Back on your initial entity type, you can select the link entity type as a potential link from your entity, and then constrain the _destination_ entities of those links by defining a selection of entity types. You can also set a minimum number of such links (e.g. to make it so that your entity type is _required_ to have certain links).
```